src/Web/AdminModule/UserModule/Control/ExternalAuthList/templates/externalAuthListControl.latte

```diff
@@ -0,0 +1,26 @@
+{templateType App\Web\AdminModule\UserModule\Control\ExternalAuthList\ExternalAuthListTemplate}
+
+<div class="px-4 sm:px-6 lg:px-8 py-4">
+    <div class="-mx-4 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+        <div class="inline-block min-w-full py-2 align-middle sm:px-6 lg:px-8">
+            <table class="min-w-full divide-y divide-gray-300">
+                <thead>
+                    <tr>
+                        <th scope="col" class="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-0">{_auth_type}</th>
+                        <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">{_resource_owner_id}</th>
+                        </th>
+                    </tr>
+                </thead>
+                <tbody class="divide-y divide-gray-200">
+                    <tr n:foreach="$externalAuths as $externalAuth">
+                        <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm text-gray-500 sm:pl-0">{$externalAuth->providerCode}</td>
+                        <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{$externalAuth->resourceOwnerId}</td>
+                    </tr>
+                    <tr n:if="0 >= count($externalAuths)">
+                        <td colspan="2" class="whitespace-nowrap py-4 pl-4 pr-3 text-sm text-gray-500 sm:pl-0">{_no_rows}</td>
```

Comment:
Can we show the "`_no_rows`" version a little bit simpler?

<img width="587" alt="Snímek obrazovky 2024-02-05 v 11 06 31" src="https://github.com/68publishers/consent-management-platform/assets/2346295/bd369613-00ad-4481-a932-6ba312839962">